CONTRIBUTING.md

```diff
@@ -99,6 +99,12 @@ This section describes the file format which is used to store the market data fo
   under a blank line in the description part. See [Git Commit Messages][git-commit-messages].
 * Check for **lint** errors.
 * Create and run **tests**.
+* For running this project locally, run an http server in the project directory  
+  For example:
+    ``` bash
+    $ python3 -m http.server
+    ```
+  You can see the website on [`http://localhost:8000`](http://localhost:8000)
```

Comment:
- I would prefer having this as a separate section because this is also useful when adding market data.
- Please capitalize `http` -> `HTTP`.
- Please connect the statements with a leading "Then you can see..".

Reply:
Added changes to address comments above :point_up: